### ISSUE TITLE
Bitz Withdraw Function Missing

### ISSUE THREAD

**Issue by drsavant-pub** [ccxt/ccxt#7911]
Question:  Is Bitz missing the _withdraw_ function purposefully, or were api withdrawals not available when Bitz was added to ccxt?

I've checked the API Docs and there is documentation for API withdrawals:  
https://apidoc.bitz.com/recharge-interface/Coin-Out.html

I also emailed Bitz customer service about the correct parameters for withdrawing usdt-tron and they emailed back: type: trc20, which confirmed for me that API withdrawals must be working.

Please advise.

- OS: win8.1
- Programming Language version: nodejs  v12.16.1
- CCXT version:  1.37.14

```
let ccxt = require('ccxt')
let exchange = new ccxt.bitz
exchange.verbose
let type = 'withdraw'

    if (exchange.has[type]){
        console.log  ('Yes! ' + exchange.name + ' has a ' + type + ' function.')
    } else {
        console.log  ('No! ' + exchange.name + ' has NO ' + type + ' function.')
    }
```

```
No! Bit-Z has NO withdraw function.
```

**Comment by kroitor** [comment]
That method is simply not implemented yet, we will add it as soon as we can, your PRs are welcome.

<img width="2766" alt="Screen Shot 2020-11-02 at 03 54 09" src="https://user-images.githubusercontent.com/1294454/97820412-16b19480-1cbf-11eb-8b8a-e98e4c4b3d6d.png">

**Comment by drsavant-pub** [comment]
Thank you!

**Comment by kroitor** [comment]
@drsavant-pub Bitz's `withdraw()` is available in CCXT 1.37.18+, the new build will arrive in 20 minutes. Don't hesitate if you have difficulties with it. Thanks!

**Comment by drsavant-pub** [comment]
Awesome!  Thanks Again.
